site/app/views/forum/ForumThreadView.php

```diff
@@ -982,7 +988,10 @@ public function createThread($category_colors){
 	public function statPage($users) {
 
 		if(!$this->forumAccess()){
-			$this->core->redirect($this->core->buildUrl(array('component' => 'navigation')));
+            $this->core->redirect($this->core->buildNewUrl([
+                $this->core->getConfig()->getSemester(),
+                $this->core->getConfig()->getCourse()
+            ]));
```

Comment:
spacing

Reply:
I thought there should be an indentation inside the function... it's like 

$this->core->redirect($this->core->buildNewUrl(*some_array*));

How would you do the spacing here?

Reply:
I made them one-line if it is preferred